gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-6/api-creation-v4-step-6.component.scss

```diff
@@ -0,0 +1,34 @@
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+.api-creation-v4-step-6 {
+  &__footer {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 8px;
+
+    &__info-icon {
+      width: 13px;
+      color: mat.get-color-from-palette(gio.$mat-space-palette, 'lighter40');
+    }
+    &__message {
+      color: mat.get-color-from-palette(gio.$mat-space-palette, 'lighter40');
+      font-weight: 700;
```

Comment:
IMHO We shouldn't have to change the font-weight. but only use typography pre configured in ui-particles 

WDYT to use : `@include mat.typography-level($typography, body-1);`